x-pack/legacy/plugins/monitoring/server/es_client/parse_elasticsearch_config.test.mocks.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const mockReadFileSync = jest.fn();
```

Comment:
Maybe we can just import this instead: https://github.com/elastic/kibana/blob/719ff259fcfc21e0b13d908227cd016157aeb19b/src/core/server/elasticsearch/elasticsearch_config.test.mocks.ts ?

Or, I guess if we import `readKeyAndCerts` from elasticsearch core we can also rely on some of its testing as well: https://github.com/elastic/kibana/blob/80b6dd8e15d0a195a6a84c2f2313902f821d022f/src/core/server/elasticsearch/elasticsearch_config.test.ts

Reply:
> Maybe we can just import this instead: https://github.com/elastic/kibana/blob/719ff259fcfc21e0b13d908227cd016157aeb19b/src/core/server/elasticsearch/elasticsearch_config.test.mocks.ts ?
I'd prefer to leave it separate since it is testing a different module.
These tests won't be needed once the Monitoring plugin moves over to the Kibana Platform, since it'll be able to use the ElasticsearchConfig and ElasticsearchClient directly.
> Or, I guess if we import `readKeyAndCerts` from elasticsearch core we can also rely on some of its testing as well: https://github.com/elastic/kibana/blob/80b6dd8e15d0a195a6a84c2f2313902f821d022f/src/core/server/elasticsearch/elasticsearch_config.test.ts
I thought about that, but that's not currently exported, and would amount to a core API change for this bug fix. I wanted to make this change as small as possible since it needs to be backported to 7.6 after the code freeze.